src/v/pandaproxy/schema_registry/storage.h

```diff
@@ -1116,6 +1116,14 @@ model::record_batch as_record_batch(Key key, Value val) {
     return std::move(rb).build();
 }
 
+template<typename Key>
+model::record_batch as_record_batch(Key key) {
```

Comment:
Seems kinda janky, or at least like it didn't already exist for a good reason. Maybe this is verboten or extremely wrong? Or better accomplished a different way?